entrypoint.sh

```diff
@@ -86,12 +91,9 @@ if [ $skipDirs ];then
     ARGS="$ARGS --skip-dirs $i"
   done
 fi
-if [ $cacheDir ];then
-  ARGS="$ARGS --cache-dir $cacheDir"
-fi
 if [ $timeout ];then
   ARGS="$ARGS --timeout $timeout"
 fi
 
 echo "Running trivy with options: " --no-progress "${ARGS}" "${artifactRef}"
-trivy ${scanType} --no-progress $ARGS ${artifactRef}
+trivy $GLOBAL_ARGS ${scanType} --no-progress $ARGS ${artifactRef}
```

Comment:
this lgtm. I'm fine with just an echo of global options on the stdout. So something like:

```
echo "Running trivy with options: " --no-progress "${ARGS}" "${artifactRef}"
echo "Global options: " "${GLOBAL_ARGS}"
```

Should be fine for now.

Reply:
@simar7 I've added the `echo` as you suggested.

Here's a sample output from running the action:

```
Running trivy with options:  --no-progress  --format  table --exit-code  1 --ignore-unfixed --vuln-type  os --severity  CRITICAL,HIGH  ghcr.io/xxx/xxx:xxx
Global options:   --cache-dir  .trivy
2021-05-27T09:20:40.443Z	INFO	Need to update DB
2021-05-27T09:20:40.443Z	INFO	Downloading DB...
2021-05-27T09:20:45.511Z	INFO	Detecting Debian vulnerabilities...

ghcr.io/xxx/xxx:xxx (debian 10.9)
===================================================================================
Total: 0 (HIGH: 0, CRITICAL: 0)
```